docs/pages/careers/developer-advocate.md

```diff
@@ -28,19 +28,19 @@ For additional details about the MUI team and culture, you can check our [career
 
 In our [last developer survey](https://mui.com/blog/2021-developer-survey-results/#what-else-can-we-do-to-improve-mui-for-you), we learned that we have a long way to go in helping developers to be successful with our technology.
 We have underinvested in this area for too long.
-Your mission will be to be the voice of the developers of the community inside the company.
-We are looking for someone that can contribute to the following outcomes:
+Your missing will be to lead the developer experience effort at MUI.
+We are looking for someone who can contribute to the following outcomes:
 
-- Create momentum in the React community and drive adoption of the library.
-- Improve the overall developer experience, resulting in better NPS & CSAT scores, e.g., through educational content.
-- Amplify the need of the community so the product direction aims at what people need the most.
+- Create excitement in the React community about the product roadmap.
+- Improve the overall developer experience, resulting in better NPS & CSAT scores, through learning content, documentation improvements, integration with other frameworks, etc.
+- Amplify the needs of the community so the product direction aims at what people need the most.
 
 Both our open-source community and our premium products are growing fast (x2 YoY).
 We need talented people to keep that going!
 
 ## About the role
 
-We are looking for a experienced developer to join us as a Developer Advocate to help the developers get familiar with and use MUI most effectively.
+We are looking for an experienced React developer to join us as a Developer Advocate to help the developers build slick UI at a speed they never experienced before, with the MUI's open-source projects.
```

Comment:
```suggestion
We are looking for an experienced React engineer to join us as a Developer Advocate to help developers build slick UI at a speed they have never experienced with MUI's open-source projects.
```

Using the word "engineer" instead of "developer" to tie back to the "DevEx engineer" title.